### PR TITLE
Reworking tutorials for local execution

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[Re-work tutorials for local services](https://trello.com/c/PgxFXTQJ/984-re-work-tutorials-for-local-services)

File: docs/ecosystem/apps/app-guide.rst [hunk 1]
```diff
@@ -107,6 +107,18 @@ For example::
...
+Local Execution
+---------------
+
+If you would like to test your application locally and it will communicate with any
+:doc`locally running services <../../getting-started/local-services>`, then you will need to
+include the ``-c path/to/config.toml`` argument when starting your application.
+
+For example::
+
+    $ ./app.py -c ../tools/config.toml
```
Comment (jacoffey3) [inline docs/ecosystem/apps/app-guide.rst]: To match the default config filename
```suggestion
    $ ./app.py -c ../tools/default_config.toml
```

File: docs/ecosystem/services/app-service.rst [hunk 2]
```diff
@@ -191,6 +191,9 @@ To manually start an application, the ``startApp`` mutation can be used.
 The mutation has two required arguments: the name of the application to start and the run level which the
 app should execute with.
 
+The optional ``config`` input argument allows a custom ``config.toml`` file to be passed to the
```
Comment (jacoffey3) [inline docs/ecosystem/services/app-service.rst]: This needs to be an absolute path.

Comment (CatGarab) [reply docs/ecosystem/services/app-service.rst]: It doesn't have to be absolute, but it's recommended. I'll add a note

**Pull request merged**